src/Serialization/main.jl

```diff
@@ -240,11 +240,13 @@ function save(filename::String, obj::Any)
 end
 
 """
-    load(io::IO)
-    load(filename::String)
+    load(io::IO; parent::Any = nothing)
+    load(filename::String; parent::Any = nothing)
 
 Load the object stored in the given io stream
 respectively in the file `filename`.
+An optional parameter `parent` can be passed, this will 
+load the file as an element of the parent.
```

Comment:
So, what are the rules: the JSON file must contain a single object at the top level, for which `parent` will be used as the parent? So if I have a `Vector` of ring elements, then this doesn't work, right? (Because we have no "parent" for `Vector` instances)